templates/fine-tune-llm_v2/README.ipynb

```diff
@@ -0,0 +1,146 @@
+    "\n",
+    "We have provided different example configurations under the `training_configs`\n",
+    "directory for different base models and accelerator types. You can use these as a\n",
+    "starting point for your own fine-tuning jobs.\n",
+    "\n",
+    "[Optional] you can get a WandB API key from [WandB](https://wandb.ai/authorize) to track the finetuning process.\n",
+    "\n",
+    "Next, you can launch a fine-tuning job where the WandB API key is passed as an environment variable.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From the VSCode terminal (press [**Ctrl + `**] in VSCode), run the command to trigger a fine-tuning job. Generally, a fine-tuning job will take a few hours.\n",
+    "\n",
+    "```bash\n",
+    "# Launch a full-param fine-tuning job for Llama 7b with 16 g5.4xlarge instances\n",
+    "$ WANDB_API_KEY={YOUR_WANDB_API_KEY} python main.py training_configs/full_param/llama-2-7b-512-16xa10.yaml\n",
+    "\n",
+    "# Launch a LoRA fine-tuning job for Llama 7b with 16 g5.4xlarge instances\n",
+    "# $ WANDB_API_KEY={YOUR_WANDB_API_KEY} python train.py training_configs/lora/llama-2-7b-512-16xa10.yaml\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Depending on whether you are running LoRA or full-param fine-tuning, you can continue\n",
+    "with step 2(a) or step 2(b)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 2(a) - Serving the LoRA finetuned model\n",
+    "\n",
+    "Upon the job completion, you can see the LoRA weight storage location and model ID in the log, such as the below:\n",
+    "\n",
+    "```shell\n",
+    "Note: LoRA weights will also be stored in path s3://anyscale-data-cld-id/org_id/cloud_id/artifact_storage/lora_fine_tuning under meta-llama/Llama-2-7b-chat-hf:sql:12345 bucket.\n",
+    "```\n",
+    "\n",
+    "You can specify this URI as the `dynamic_lora_loading_path` in the serving workspace template, and then query the endpoint.\n",
+    "\n",
+    "Note: Model IDs follow the format `{base_model_id}:{suffix}:{id}`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 2(b) - Serving the full-param finetuned model\n",
+    "\n",
+    "Once the fine-tuning job is complete, you can view the stored full-param fine-tuned model weight at the very end of the job logs. Here is an example finetuning job output:\n",
+    "\n",
+    "```shell\n",
+    "Best checkpoint is stored in:\n",
+    "anyscale-data-cld-id/org_id/cloud_id/artifact_storage/username/llmforge-finetuning/meta-llama/Llama-2-70b-hf/TorchTrainer_2024-01-25_18-07-48/TorchTrainer_b3de9_00000_0_2024-01-25_18-07-48/checkpoint_000000\n",
+    "```\n",
+    "\n",
+    "You can follow the `Learn how to bring your own models` section under the endpoints template to serve this finetuned model with the specified storage uri."
```

Comment:
Same here, link. Also, don't use the term "endpoints" as that is being deprecated.

Reply:
Updated